src/main/java/webdriver/screen/yandex/disk/YandexDiskPage.java

```diff
@@ -32,28 +37,44 @@ private By createLocatorForFilesByName(String name) {
 
     public YandexDiskPage createNewFolder(String nameFolder) {
         browser.clickOnButton(BUTTON_CREATE_LOCATOR);
+        logger.info("Clicked on create button");
```

Comment:
createNewFolder, moveFileInTresh, createFolderFromContextMenu, and such other methods should be performed on a service layer. Please consider it for the next homework